### PR TITLE
Bootstrapify

### PR DESCRIPTION
**Pull request by crookedneighbor**
OLD: 

screen shot 2016-04-01 at 4 43 59 pm

New:

screen shot 2016-04-01 at 4 43 49 pm

### CODE REVIEW

**Comment by crookedneighbor** [comment]
@wjr1985 If you're :+1: on these changes, I can change the rest of the pages to bootstrapify them

**Comment by wjr1985** [comment]
:+1: this looks great @crookedneighbor!

**Comment by crookedneighbor** [comment]
This should be ready to merge now.

File: views/spin.haml [hunk 1]
```diff
@@ -9,18 +9,22 @@
...
-        %input(type="checkbox" name="animate" value="true") Animate the image (if it's a GIF)
-        %br
-        %input(type="checkbox" name="resize" value="32x32") Resize* the image to attempt to make the filesize smaller.
-        %br
-        %input(type="submit" value="GOGOGOGOGOGOGOGO")
-      %br
-      * The image will be resized to 32x32, but the aspect ratio will be retained, so you may up with an image with dimensions 32x12 or 10x32 for example.
+        .form-group.form-inline
+          %select#rotations.form-control(type="number" name="rotations")
+            - (1..50).each do |i|
```
Comment (crookedneighbor) [inline views/spin.haml:18]: Made this a select box because it was easier to style

Comment (wjr1985) [inline views/spin.haml:18]: 👍

**Pull request merged**